svc/wallet/transport.go

```diff
@@ -87,7 +87,7 @@ func MakeHTTPHandler(e Endpoints, log logger) http.Handler {
 		options...,
 	).ServeHTTP)
 
-	r.Get("/{wallet_id}/stake", httptransport.NewServer(
+	r.Get("/stake", httptransport.NewServer(
```

Comment:
pls don't change HTTP interface, this functionality is already implemented in the mobile app